### DISCUSSION TITLE
Github Education Application approved but not receive my academic benefits

### DISCUSSION THREAD

**Discussion by stdcat** [community/community#132032] [GitHub Education]
### Hi everyone,

Hello I was verified with my academic details and was approved on July 5, still not get the benefits of student developer pack its been more than 1 week.
![image](https://github.com/user-attachments/assets/fccd0e4f-2af3-4c29-829a-7bc58fc6c716)

**Comment by Adya-Prasad** [comment]
Hi stdcat
I can understand your concern!
I've included the section that I feel would best assist.
 [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
 [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)

If You still have any concerns! Mark my comment as an answer and reply to me with your concern. I would love to help you ahead

**Comment by samus-aran** [comment]
I'll be closing this discussion due to lack of recent activity. We can confirm that you have since been successful - enjoy your journey through GitHub Education.
